docker-compose.open.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.8'
```

Comment:
With all these compose files, some comments in each explaining their purpose and what they do might be helpful.

Reply:
√